routes/web.php

```diff
@@ -24,46 +24,163 @@
 */
 
 Route::get('/', function () {
-  return redirect()->route('login');
+    return redirect()->route('login');
 });
 
 Route::middleware(['auth:sanctum', 'verified'])->group(function () {
-  Route::get('/dashboard', function () {
-    return Inertia\Inertia::render('Dashboard');
-  })->name('dashboard');
-
-  Route::name('admin.')->prefix('admin')->group(function () {
-    //TODO: Admin index route
-    Route::resource('users', UserController::class)->only(['store', 'index', 'destroy', 'update']);
-    Route::resource('roles', RoleController::class)->except(['create', 'show', 'edit']);
-
-    Route::resource('rooms', RoomController::class)->only(['store', 'index', 'update', 'destroy']);
-    Route::put('rooms/{room}/restrictions', [RestrictionsController::class, 'update'])
-      ->name('room.restrictions.update')->middleware('permission:bookings.approve');
-    Route::put('rooms/{room}/date-restrictions', RoomDateRestrictionsController::class)
-      ->name('room.date.restrictions.update')->middleware('permission:bookings.approve');
-    Route::resource('rooms.blackouts', BlackoutController::class)->only(['index', 'store', 'destroy', 'update']);
-
-    Route::name('settings.')->prefix('settings')->group(function () {
-      Route::get('/', [SettingsController::class, 'index'])->name('index');
-      Route::post('app_logo', [SettingsController::class, 'storeAppLogo'])->name('app.logo');
-      Route::post('app_name', [SettingsController::class, 'storeAppName'])->name('app.name');
+    Route::get('/dashboard', function () {
+        return Inertia\Inertia::render('Dashboard');
+    })->name('dashboard');
+
+    /**
+     * ADMIN
+     */
+    Route::name('admin.')->prefix('admin')->group(function () {
+        //TODO: Admin index route
+
+        /**
+         * USERS
+         */
+        Route::name('users.')->prefix('users')->group(function() {
+            Route::get('/', [UserController::class, 'index'])
+                ->name('index');
+
+            Route::post('/', [UserController::class, 'store'])
+                ->name('store')
+                ->middleware(['permission:users.create']);
+
+            Route::put('/{user}', [UserController::class, 'update'])
+                ->name('update')
+                ->middleware(['permission:users.update']);
+
+            Route::delete('/{user}', [UserController::class, 'destroy'])
+                ->name('destroy')
+                ->middleware(['permission:users.delete']);
```

Comment:
Is this the responsibility of the admin to create users or do the users create themselves?

Reply:
As per stakeholder discussion, user creation will be manual